storage/objects/main.go

```diff
@@ -424,6 +426,75 @@ func downloadUsingRequesterPays(client *storage.Client, object, bucketName, loca
 	return nil
 }
 
+func generateV4GetObjectSignedUrl(client *storage.Client, bucketName, objectName, serviceAccount string) (string, error) {
+	// [START storage_generate_signed_url_v4]
+	jsonKey, err := ioutil.ReadFile(serviceAccount)
+	if err != nil {
+		return "", fmt.Errorf("cannot read the JSON key file, err: %v", err)
+	}
+
+	conf, err := google.JWTConfigFromJSON(jsonKey)
+	if err != nil {
+		return "", fmt.Errorf("google.JWTConfigFromJSON: %v", err)
+	}
+
+	opts := &storage.SignedURLOptions{
+		Scheme: storage.SigningSchemeV4,
+		Method: "GET",
+		GoogleAccessID: conf.Email,
+		PrivateKey:     conf.PrivateKey,
+		Expires:        time.Now().Add(15*time.Minute),
+	}
+
+	u, err := storage.SignedURL(bucketName, objectName, opts)
+	if err != nil {
+		return "", fmt.Errorf("Unable to generate a signed URL: %v", err)
+	}
+
+	fmt.Printf("Generated GET signed URL:\n");
```

Comment:
No semicolons needed at the ends of the lines. :)

Reply:
lol... I can Go, I promise! Thanks for catching those.